lib/svg4everybody.js

```diff
@@ -0,0 +1,95 @@
+/*! svg4everybody v1.0.0 | github.com/jonathantneal/svg4everybody */
+
+var CACHE = {};
+var NOSVG = /MSIE\s[1-8]\b/.test(navigator.userAgent);
+var NOEXT = NOSVG || (navigator.userAgent.match(/AppleWebKit\/(\d+)|Edge\/12\b|Trident\/[567]\b/) || [])[1] < 538;
+
+if (NOSVG) {
+	document.createElement('svg');
+	document.createElement('use');
+}
+
+var uses = document.getElementsByTagName('use');
+var requestAnimationFrame = window.requestAnimationFrame || window.setTimeout;
```

Comment:
If you are wrapping this with UMD you can save the `window.` keyword, as well as `document` and `navigator`, as the context will always fallback to the correct value given the environment. (`window`|`process`)